ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -942,17 +942,16 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
           </WorkspaceEditSection>}
         <hr style={{marginTop: '1rem'}}/>
         <WorkspaceEditSection header='Research Use Statement Questions'
-              description={<div style={{marginLeft: '-0.9rem', fontSize: 14}}> {ResearchPurposeDescription}
+              description={<div style={{marginLeft: '-0.9rem', fontSize: 14,
+                backgroundColor: colorWithWhiteness(colors.accent, 0.85)}}>
+                <div style={{margin: '0.5rem', paddingTop: '0.5rem'}}>{ResearchPurposeDescription}
               <br/><br/>
-              <i>Note that you are required to create separate Workspaces for each project
-                for which you access All of Us data, hence the responses below are expected
-                to be specific to the project for which you are creating this particular
-                Workspace.</i></div>
+              </div></div>
             }/>
 
         {/*Primary purpose */}
         <WorkspaceEditSection header={researchPurposeQuestions[0].header}
-            description={researchPurposeQuestions[0].description} index='1.' indent>
+            description={researchPurposeQuestions[0].description} index='1.' indent publiclyDisplayed>
```

Comment:
nitpick: can you do this by saying `indent={true} publiclyDisplayed={true}` instead ?